gordo_components/builder/build_model.py

```diff
@@ -88,6 +90,13 @@ def build_model(
     -------
         Tuple[Optional[sklearn.base.BaseEstimator], dict]
     """
+    if evaluation_config.get("seed"):
+        seed = evaluation_config["seed"]
+        logger.info(f"Setting random seed: '{seed}'")
+        tf.random.set_seed(seed)
+        np.random.seed(seed)
+        random.seed(seed)
```

Comment:
Would it be worth testing that these all infact got set to `seed`?  Maybe if these libs change how things get set, and these calls don't act as planned in a way that wouldn't affect the `df1==df2` test but could affect some other pipeline not represented in the tests.

Reply:
It would be _very_ strange indeed if the api changed in such a way the seed was _not_ set and yet the test as it is, still passed. 

anyhow, according to _some guy_, it doesn't seem (that) possible. 
[The short answer is that you simply can't (at least not in general).](https://stackoverflow.com/questions/32172054/how-can-i-retrieve-the-current-seed-of-numpys-random-number-generator)

Reply:
Right on :+1:  Sounds good me. The comments in *some dude"'s answer got *heated*.